### PR TITLE
Add linting to tests

### PR DESCRIPTION
**Pull request by miguelpduarte**
- Complements #28, this was probably forgotten or would have been
included there
- Fix linting errors. This was mostly automatic but there was also some
manual work relative to following mocha's suggestions for using
async/await in tests for the auth tests
(https://mochajs.org/#using-async-await)

### CODE REVIEW

File: test/common.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-//// Base file for unit test utilities
+// // Base file for unit test utilities
```
Comment (xRuiAlves) [inline test/common.js]: Double comment makes the compiler ignore this twice, right? 😂

File: test/sample_tests.js [hunk 2]
```diff
@@ -20,18 +20,19 @@ describe("# Sample boilerplate tests", () => {
         });
```
Comment (xRuiAlves) [inline test/sample_tests.js]: This file could be probabbly removed by now, right?

Comment (miguelpduarte) [reply test/sample_tests.js]: Out of the scope of this PR, imo

Leaving it for #19 - that would be more inside scope and there will be a better space for discussion and thought on the removal of it

File: test/common.js (hunk 1 above)
Comment (miguelpduarte) [reply test/common.js]: :man_facepalming: 

Thanks eslint!

Couldn't change the rules so just added moar comments :slightly_smiling_face:

**Pull request merged**